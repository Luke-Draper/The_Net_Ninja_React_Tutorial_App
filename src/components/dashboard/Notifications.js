import React, { Component } from "react";

class Notifications extends Component {
	render() {
		return (
			<div className="notifications">
				<h3>Notifications</h3>
			</div>
		);
	}
}

export default Notifications;
