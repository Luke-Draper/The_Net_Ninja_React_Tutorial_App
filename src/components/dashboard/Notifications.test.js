import React from "react";
import Notifications from "./Notifications";
import { MemoryRouter } from "react-router-dom"; //Required when component uses a Route or Link
import renderer from "react-test-renderer";

test("Notifications renders without crashing", () => {
	const component = renderer.create(
		<MemoryRouter initialEntries={["/"]}>
			<Notifications />
		</MemoryRouter>
	);
	expect(component.toJSON()).toMatchSnapshot();
});
