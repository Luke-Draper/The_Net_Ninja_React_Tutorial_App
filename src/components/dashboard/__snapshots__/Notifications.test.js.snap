// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notifications renders without crashing 1`] = `
<div
  className="Notifications"
>
  <p>
    Notifications
  </p>
</div>
`;
